src/compas/scene/scene.py

```diff
@@ -117,10 +117,21 @@ def add(self, item, parent=None, **kwargs):
                 if kwargs["context"] != self.context:
                     raise Exception("Object context should be the same as scene context: {} != {}".format(kwargs["context"], self.context))
                 del kwargs["context"]  # otherwist the SceneObject receives "context" twice, which results in an error
+            if isinstance(parent, Group):
+                # Use the kwargs of the parent group as default values for the child sceneobject
+                group_kwargs = parent.kwargs.copy()
+                group_kwargs.update(kwargs)
+                kwargs = group_kwargs
             sceneobject = SceneObject(item=item, context=self.context, **kwargs)  # type: ignore
         super(Scene, self).add(sceneobject, parent=parent)
         return sceneobject
 
+    def add_group(self, name, parent=None, **kwargs):
+        # type: (str, SceneObject | TreeNode | None, dict) -> SceneObject
+        group = Group(name=name, **kwargs)
+        self.add(group, parent=parent)
+        return group
+
```

Comment:
align with viewer. always recommend to use this instead of creating the group by yourself